__tests__/e2e.test.ts

```diff
@@ -192,20 +192,10 @@ describe('End to end tests', () => {
    * - Delete script and verify it is deleted in state file
    */
   test('✅ Script features', async () => {
-    // Arrange: Setup init
-    commands.setupCommands(program);
-
-    // Act
-    await program.parseAsync(['node', 'hedera-cli.ts', 'setup', 'init']);
-
-    // Assert
-    let accounts = stateController.get('accounts');
-    expect(accounts['testnet-operator']).toBeDefined();
-
     // Arrange: Download a script from the internet
     commands.stateCommands(program);
     const scriptURL =
-      'https://raw.githubusercontent.com/hashgraph/hedera-cli/main/src/commands/script/examples.json';
+      'https://raw.githubusercontent.com/hashgraph/hedera-cli/0e724e64f3c95c41ec4a8765e3a58733891328ea/src/commands/script/examples.json';
```

Comment:
Why should we opt for a specific commit hash instead of the main branch?

Reply:
@michielmulders The current `main` branch file forces the use of the Test Network and until this PR is merged we cannot depend on the `example.json` from `main`.

Reply:
Good!